iroha-cli/interactive/impl/interactive_common_cli.cpp

```diff
@@ -87,12 +97,20 @@ namespace iroha_cli {
       return line;
     }
 
+    boost::optional<std::string> promptString(ParamData param) {
+        std::string message = std::get<0>(param);
+        if (not std::get<1>(param).empty()) {
```

Comment:
std::accumulate is also applicable here

Reply:
Don't think it is appropriate to use `std::accumulate` here. `param` is just a `std::pair` (and changing to a struct).

Reply:
Oops, indeed. I though there was a loop, nvm then